### PR TITLE
feat(chip): add color support

### PR DESCRIPTION
**Pull request by matmkian**

### CODE REVIEW

**Comment by matmkian** [comment]
To do in react version

File: src/components/chip/angularjs/chip_directive.js [hunk 1]
```diff
@@ -121,6 +122,24 @@ function ChipDirective() {
         if (transclude.isSlotFilled('label')) {
             ctrl.hasLabel = true;
         }
+
+        const defaultProps = {
+            color: 'dark',
```
Comment (arnaudvalle) [inline src/components/chip/angularjs/chip_directive.js]: Just to confirm, `dark` is a color or a theme? Looking at the template, it looks like you wanted to make the default theme `dark`?

File: src/components/chip/react/__snapshots__/Chip.test.tsx.snap [hunk 2]
```diff
@@ -3,7 +3,7 @@
 exports[`<Chip /> Snapshot should render correctly Chip component 1`] = `
 <a
   LabelComponent="Hello World!"
-  className="lumx-chip lumx-chip--size-m lumx-chip--theme-light lumx-chip--is-unselected"
+  className="lumx-chip lumx-chip--color-dark lumx-chip--size-m lumx-chip--is-unselected"
```
Comment (arnaudvalle) [inline src/components/chip/react/__snapshots__/Chip.test.tsx.snap]: So no theme by default is actually what we want?

File: src/components/chip/angularjs/chip_directive.js (hunk 1 above)
Comment (mcastagnetti) [reply src/components/chip/angularjs/chip_directive.js]: Don't know about angular version, @matmkian ?

File: src/components/chip/react/__snapshots__/Chip.test.tsx.snap (hunk 2 above)
Comment (mcastagnetti) [reply src/components/chip/react/__snapshots__/Chip.test.tsx.snap]: Yep :3

File: src/components/chip/angularjs/chip_directive.js (hunk 1 above)
Comment (matmkian) [reply src/components/chip/angularjs/chip_directive.js]: No more default value for theme prop
Add color prop with dark default value -> dark is a theme but also a color :-)

Comment (arnaudvalle) [reply src/components/chip/angularjs/chip_directive.js]: :wtfception:

**Comment by matmkian** [comment]
Test ok!

**Pull request merged**